<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DRecorder</class>
 <widget class="QWidget" name="DRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DRecorder</string>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>recorder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DRecorder录音机</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置文件名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>录音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提示：
         录音文件被保存在用户主目录（即桌面上的 主文件夹）</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
